apps/marginfi-v2-trading/src/components/common/admin/admin-pool-detail/pool-detail-card/pool-detail-card.tsx

```diff
@@ -0,0 +1,364 @@
+          <dt>Deposit/Borrow APY</dt>
+          <dd className="text-right text-primary">
+            <span className="text-mrgn-success">{percentFormatter.format(lendingRate.rateAPY)}</span>
+            {" / "}
+            <span className="text-mrgn-error">{percentFormatter.format(borrowingRate.rateAPY)}</span>
+          </dd>
+          <dt>Deposits</dt>
+          <TooltipProvider>
+            <Tooltip>
+              <TooltipTrigger asChild>
+                <dd className="text-right text-primary flex items-center gap-1 justify-end">
+                  {numeralFormatter(bankDepositData.bankDeposits)} {bank.meta.tokenSymbol}
+                  {bankDepositData.isBankHigh || bankDepositData.isBankFilled ? (
+                    <IconAlertTriangle size={14} />
+                  ) : (
+                    <IconInfoCircle size={14} />
+                  )}
+                </dd>
+              </TooltipTrigger>
+              <TooltipContent className="text-left flex flex-col gap-2">
+                {bankDepositData.isBankHigh && (
+                  <div>{bankDepositData.isBankFilled ? "Limit Reached" : "Approaching Limit"}</div>
+                )}
+
+                <span>
+                  {bankDepositData.symbol} {bankDepositData.isInLendingMode ? "deposits" : "borrows"} are at{" "}
+                  {percentFormatter.format(bankDepositData.capacity)} capacity.
+                </span>
+                {!bankDepositData.isBankFilled && <span>Available: {numeralFormatter(bankDepositData.available)}</span>}
+                <a href="https://docs.marginfi.com">
+                  <u>Learn more.</u>
+                </a>
+              </TooltipContent>
+            </Tooltip>
+          </TooltipProvider>
+          <dt>Borrows</dt>
+          <TooltipProvider>
+            <Tooltip>
+              <TooltipTrigger asChild>
+                <dd className="text-right text-primary flex items-center gap-1 justify-end">
+                  {numeralFormatter(bankCapBorrowing.bankCap)} {bank.meta.tokenSymbol}
+                  {bankBorrowData.isBankHigh || bankBorrowData.isBankFilled ? (
+                    <IconAlertTriangle size={14} />
+                  ) : (
+                    <IconInfoCircle size={14} />
+                  )}
+                </dd>
+              </TooltipTrigger>
+              <TooltipContent className="text-left flex flex-col gap-2">
+                {bankBorrowData.isBankHigh && (
+                  <div>{bankBorrowData.isBankFilled ? "Limit Reached" : "Approaching Limit"}</div>
+                )}
+
+                <span>
+                  {bankBorrowData.symbol} {bankBorrowData.isInLendingMode ? "deposits" : "borrows"} are at{" "}
+                  {percentFormatter.format(bankBorrowData.capacity)} capacity.
+                </span>
+                {!bankBorrowData.isBankFilled && <span>Available: {numeralFormatter(bankBorrowData.available)}</span>}
+                <a href="https://docs.marginfi.com">
```

Comment:
More of a general comment; there are a lot of hrefs to the docs. We should have a process to href to the correct section in the docs. If there isnt a corresponding section, means that our docs should be updated imo